packages/grafana-prometheus/src/querybuilder/components/metrics-modal/state/helpers.ts

```diff
@@ -89,6 +89,20 @@ function buildMetricData(metric: string, datasource: PrometheusDatasource): Metr
   return metricData;
 }
 
+export function getMetadataHelp(metric: string, metadata: PromMetricsMetadata): string | undefined {
+  if (!metadata[metric]) {
+    return undefined;
+  }
+  return metadata[metric].help;
```

Comment:
nit: I think you could simplify this a bit:
```suggestion
  return metadata[metric]?.help;
```